cmake/FindFreetype.cmake

```diff
@@ -70,6 +70,7 @@ set(FREETYPE_FIND_ARGS
     ENV GTKMM_BASEPATH
     [HKEY_CURRENT_USER\\SOFTWARE\\gtkmm\\2.4;Path]
     [HKEY_LOCAL_MACHINE\\SOFTWARE\\gtkmm\\2.4;Path]
+	$ENV{FREETYPE_DIR}
```

Comment:
`FREETYPE_DIR` is already a path hint a few lines above, why do we need to duplicate it? According to the CMake manual, hints and paths are the same, hints are just searched before paths. Also tabs vs. spaces.

Reply:
indeed. not necessary.